components/admin_console/team_channel_settings/team/list/__snapshots__/team_list.test.tsx.snap

```diff
@@ -134,6 +134,7 @@ exports[`admin_console/team_channel_settings/team/TeamList should match snapshot
                 >
                   DN
                 </b>
+                
```

Comment:
seems like these spaces are coming from a diff formatter settings can we remove them?